src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -678,6 +678,10 @@ public static EJBClientContext requireCurrent() {
     <T> StatefulEJBLocator<T> createSession(final StatelessEJBLocator<T> statelessLocator, final AuthenticationConfiguration authenticationConfiguration, final SSLContext sslContext) throws Exception {
         final LocatedAction<StatefulEJBLocator<T>, StatelessEJBLocator<T>, T> action =
             (receiver, originalLocator, newAffinity, ac, sc) -> receiver.createSession(originalLocator.withNewAffinity(newAffinity), ac, sc);
+
+        if (Logs.INVOCATION.isDebugEnabled()) {
+            Logs.INVOCATION.debugf("Calling createSession(module = %s, strong affinity = %s): ", ((EJBLocator)statelessLocator).getIdentifier(), statelessLocator.getAffinity());
```

Comment:
This is a per-invocation log message, so it has to be at level TRACE.  Also, you can just log the locator as a string parameter since a locator is just a combination of module identifier and affinity (plus the view type, which is not harmful to include in a log message).  Because no interpolation is required to perform the log statement, you don't need the enclosing ```is*Enabled``` stuff.